sup3r/models/data_centric.py

```diff
@@ -137,6 +138,71 @@ def calc_temporal_losses(total_losses, content_losses, batch_handler):
                     f'{round_array(new_temporal_weights)}')
 
 
+class WindGanDC(WindGan, Sup3rGanDC):
```

Comment:
nice and clean subclass. A sign of good design!

Reply:
Thanks!